main.css

```diff
@@ -0,0 +1,602 @@
+    border: 1px solid #EFEFEF;
+    box-sizing: border-box;
+    border-radius: 13px;
+    min-width: 480px;
+    padding: 13px 18px;
+    margin-bottom: 18px;
+}
+
+.footer-input-msg {
+    color: #7C7C7C;
+    border: 1px solid #EFEFEF;
+    box-sizing: border-box;
+    border-radius: 13px;
+    min-width: 480px;
+    min-height: 94px;
+    padding: 13px 18px 83px;
+    margin-bottom: 18px;
+}
+
+.footer-input-button {
+    background: #006AFF;
+    padding: 13px 39px !important;
+    margin-bottom: 40px;
+    color: #fff;
+}
+
+.footer-copyright {
+    text-align: center;
+    color: #141516;
+    padding: 20px 0;
+    margin: 0;
+    border-top: #C9C9C9 1px solid;
+}
+
+body {
+    margin: 0;
+    padding: 0;
+}
+
+ul {
+    list-style-type: none;
+    margin: 0;
+    padding: 0;
+}
+
+.hero-section {
+    background-color: #141516;
+    justify-content: center;
+    display: flex;
+}
+
+.hero-flex {
+    width: 1140px;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+}
+
+.hero-nav {
+    padding-top: 35px;
+    display: flex;
+    justify-content: space-between;
+    color: #fff;
+}
+
+.logo-box {
+    padding-right: 96px;
+}
+
+.hero-logo {
+    cursor: pointer;
+}
+
+.nav-left-content {
+    display: flex;
+    justify-content: center; 
+}
+
+.nav-right-content {
+    display: flex;
+}
+
+.nav-ul {
+    display: flex;
+    justify-content: space-between;
+    min-width: 340px;
+}
+
+.right-nav-ul {
+    display: flex;
+}
+
+.nav-link {
+    margin: 0;
+}
+
+.nav-button {
+    background: #006AFF;
+    color: #fff;
+    margin-top: -13px;
+    margin-left: 56px;
+}
+
+.hero-title {
+    color: #fff;
+    max-width: 500px;
+    padding-top: 58px;
+    margin: 0;
+}
+
+.hero-para {
+    color: #fff;
+    max-width: 440px;
+    margin-bottom: 50px;
+}
+
+.hero-cta-button {
+    background: #006AFF;
+    color: #fff;
+}
+
+.hero-login-cta-button{
+    color: #fff;
+    border: 1px solid #fff !important;    
+    padding: 13px 61px !important;
+    background-color: #141516;
+}
+
+.hero-cta {
+    display: flex;
+    justify-content: space-between;
+    max-width: 360px;
+    align-items: center;
+}
+
+.hero-content {
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 135px;
+    padding: 100px 0 0;
+}
+
+.hero-content-img {
+    position: relative;
+    padding: 0 35px 0 15px;
+}
+
+.hero-img-phone {
+    position: absolute;
+    bottom: 0;
+    left: 0;
+}
+
+.hero-live-box {
+    background-color: #fff;
+    height: 30px;
+    width: 72px;
+    border-radius: 20px;
+    z-index: 1;
+    position: absolute;
+    top: -21px;
+    right: 0px;
+    align-items: center;
+    justify-content: center;
+    display: flex;
+}
+
+.hero-platforms-list {
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 74px;
+}
+
+.button {
+    border-radius: 25px;
+    border: 0;
+    cursor: pointer;
+    padding: 13px 29px;
+    font-family: 'Mulish', sans-serif;
```

Comment:
w zasadzie to mozesz ustawic globalnie font-family i stosowac inny kiedy to jest potrzebne jak w tej klasie